### PR TITLE
Updated documentation on setting Files as the default file manager

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: src/routes/docs/configuring/replace-file-explorer.svx [hunk 1]
```diff
@@ -1,24 +1,11 @@
-# Replacing File Explorer with Files (Unsupported)
+# Replacing File Explorer with Files (Experimental)
 
-*This guide involves modifying the registry, make sure to create a backup beforehand and proceed at your own risk.
-Please keep in mind that this method is unsupported and may not work for everyone.*
+*This setting modifies the system registry, make sure to create a backup beforehand and proceed at your own risk.
+Please keep in mind that these changes are not reverted when removing the app so make sure to turn this setting off before removing the app from your device.*
```
Comment (chingucoding) [inline src/routes/docs/configuring/replace-file-explorer.svx:4]: Out of curiosity, does installing Files again make it possible to revert this setting?

**Review by chingucoding — APPROVED** [review]
LGTM

File: src/routes/docs/configuring/replace-file-explorer.svx (hunk 1 above)
Comment (yaichenbaum) [reply src/routes/docs/configuring/replace-file-explorer.svx:4]: Yes

Comment (chingucoding) [reply src/routes/docs/configuring/replace-file-explorer.svx:4]: Good to know, thanks!

**Pull request merged**